internal/protoschema/protoschema.go

```diff
@@ -16,5 +16,5 @@ package protoschema
 
 const (
 	// Version is the current version of protoschema which is shared by all plugins.
-	Version = "0.1.0"
+	Version = "0.2.0-dev"
```

Comment:
I don't remember if we bump this to the next minor or patch. Probably doesn't matter and I'm not sure we're consistent across our repositories.